go.mod

```diff
@@ -85,3 +85,5 @@ require (
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/livekit/protocol => ../protocol
```

Comment:
Will clean this up. The corresponding protocol PR (and some discussions with Shishir) is in this PR - https://github.com/livekit/protocol/pull/86